src/ansys/mapdl/core/_commands/post1/_fatigue.py

```diff
@@ -0,0 +1,652 @@
+        automatically defined for a node not having a location when the FSSECT, FSNODE, or FS command is
+        issued. Automatically defined locations are assigned the lowest available location number, unity
+        stress concentration factors, and no title.
+
+        .. warning::
+
+            This command is archived in the latest version of the software.
+
+        """
+        command = f"FL,{nloc},{node},{scfx},{scfy},{scfz},{title}"
+        return self.run(command, **kwargs)
+
+    def fllist(self, nloc1: str = "", nloc2: str = "", ninc: str = "", **kwargs):
+        r"""Lists the fatigue location parameters.
+
+        Mechanical APDL Command: `FLLIST <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_FLLIST.html>`_
+
+        Parameters
+        ----------
+        nloc1 : str
+            List location parameters from ``NLOC1`` (defaults to 1) to ``NLOC2`` (defaults to ``NLOC1`` ) in
+            steps of ``NINC`` (defaults to 1). If ``NLOC1`` = ALL, ``NLOC2`` and ``NINC`` are ignored and
+            all locations are listed.
+
+        nloc2 : str
+            List location parameters from ``NLOC1`` (defaults to 1) to ``NLOC2`` (defaults to ``NLOC1`` ) in
+            steps of ``NINC`` (defaults to 1). If ``NLOC1`` = ALL, ``NLOC2`` and ``NINC`` are ignored and
+            all locations are listed.
+
+        ninc : str
+            List location parameters from ``NLOC1`` (defaults to 1) to ``NLOC2`` (defaults to ``NLOC1`` ) in
+            steps of ``NINC`` (defaults to 1). If ``NLOC1`` = ALL, ``NLOC2`` and ``NINC`` are ignored and
+            all locations are listed.
+
+        Notes
+        -----
+
+        .. _FLLIST_notes:
+
+        No additional usage notes are available for this command.
+
+        .. warning::
+
+            This command is archived in the latest version of the software.
+
+        """
+        command = f"FLLIST,{nloc1},{nloc2},{ninc}"
+        return self.run(command, **kwargs)
+
+    def fp(
+        self,
+        stitm: int | str = "",
+        c1: str = "",
+        c2: str = "",
+        c3: str = "",
+        c4: str = "",
+        c5: str = "",
+        c6: str = "",
+        **kwargs,
+    ):
+        r"""Defines the fatigue S vs. N and Sm vs. T tables.
+
+        Mechanical APDL Command: `FP <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_FP.html>`_
+
+        Parameters
+        ----------
+        stitm : int or str
+            Starting item number for entering properties (defaults to 1). If 1, data input in field ``C1`` of this command is entered as the first item in the list; if 7, data input in field ``C1`` of this command is entered as the seventh item in the list; etc. If the item number is negative, ``C1`` - ``C6`` are ignored and the item is deleted. If -ALL, the table is erased. Items are as follows (items 41-62 are required only if simplified elastic-plastic code calculations are to be performed):
```

Comment:
Very long line.

Reply:
Oh... it seems it should be a list of arguments on itself

Reply:
It does render well though... (the latest `62` does not though... the extra commma seems an issue.

<img width="867" alt="image" src="https://github.com/user-attachments/assets/efb84d1e-bea7-43b7-bac4-dc3f258a11fd" />

Reply:
So I guess the issue is just the line length really.